android/src/main/java/com/bitmovin/player/reactnative/SourceModule.kt

```diff
@@ -29,16 +31,10 @@ class SourceModule(private val context: ReactApplicationContext) : ReactContextB
     override fun getName() = MODULE_NAME
 
     /**
-     * Fetches the `Source` instance associated with `nativeId` from internal sources.
-     * @param nativeId `Source` instance ID.
-     * @return The associated `Source` instance or `null`.
+     * Fetches the [Source] instance associated with [nativeId] from internal sources or throws.
      */
-    fun getSource(nativeId: NativeId?): Source? {
-        if (nativeId == null) {
-            return null
-        }
-        return sources[nativeId]
-    }
+    fun getSource(nativeId: NativeId): Source = sources[nativeId]
```

Comment:
Add a receiver `throwSafeContext` to make sure this is not called in a context that would crash the app?

Reply:
Done: b39491e050674d0a134c35a1e9690c90872eebc2 (it wasn't very complicated.)